Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -328,12 +361,12 @@ extension GoogleSheetsManager {
 
 // MARK: Writing to Google Sheets
 extension GoogleSheetsManager {
-  func addTransaction(amount: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int, completion: @escaping (Bool) -> Void) {
+  func addTransaction(amount: String, memo: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int, completion: @escaping (Bool) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/67edf3325570da301c03450fdbbe6d8b96d1b452/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L364"></span>Function should have 5 parameters or less: it currently has 8<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:364</code></td>
    </tr>
  </tbody>
</table>